maml/apps/pes/_mtp.py

```diff
@@ -105,8 +105,8 @@ def _line_up(self, structure, energy, forces, virial_stress):
             lines.append("{:>24.12f}".format(inputs["Energy"]))
         if "Stress" in inputs:
             format_str = "{:>12s}{:>12s}{:>12s}{:>12s}{:>12s}{:>12s}"
-            format_float = "{:>12f}{:>12f}{:>12f}{:>12f}{:>12f}{:>12f}"
-            lines.append(format_str.format("Stress:  xx", "yy", "zz", "yz", "xz", "xy"))
+            format_float = "{:>16f}{:>12f}{:>12f}{:>12f}{:>12f}{:>12f}"
+            lines.append(format_str.format("PlusStress:  xx", "yy", "zz", "yz", "xz", "xy"))
```

Comment:
The notation of Stress in the .cgf file, which stores training data for MTP, has changed to "PlusStress" in mlip-2.